src/Platform/Microsoft.Testing.Platform/Hosts/TestHostBuilder.cs

```diff
@@ -195,6 +196,8 @@ internal class TestHostBuilder(IFileSystem fileSystem, IRuntimeFeature runtimeFe
         // Add the platform output device to the service provider for both modes.
         serviceProvider.TryAddService(platformOutputDevice);
 
+        CommandLine.AddProvider(() => new TerminalTestReporterCommandLineOptionsProvider());
```

Comment:
@MarcoRossignoli please validate the position of this config

Reply:
I think it's fine but we should add a comment with a TODO we're adding this one to avoid the failure for unknown options but the current output is using "raw" parsing, as we discussed for now it's fine, we should proxy the `ICommandLineOptions` to avoid this I'd like to have a ticket

Reply:
we are discussing it in the chat, I won't forget to fix it  however we decide to do it.

https://github.com/microsoft/testfx/issues/3552